### DISCUSSION TITLE
disable reporting to terminal serach result

### DISCUSSION THREAD

**Discussion by Ahmad-Zaklouta** [vscode-neovim/vscode-neovim#2083] [Q&A] [answered]
Hey!

when I do a search and the pattern is not in the file, a terminal is open with the message:

  **search hit BOTTOM, continuing at TOP
  E486: Pattern not found: \<LLSURX_fifo_reg\>**

and I have to close the terminal every time.

Is there a way to disable reporting to the terminal?

**Comment by ollien** [comment]
@Ahmad-Zaklouta what version are you running? This should be fixed with 1.15

See: https://github.com/vscode-neovim/vscode-neovim/issues/2046

**Comment by ollien** [comment, marked as answer]
Aha! Someone pointed something out here. This should be added to the [docs shortly](https://github.com/vscode-neovim/vscode-neovim/pull/2092) but you can add `set shortmess+=s` to your `init.vim`
